frontend/src/utils.js

```diff
@@ -48,19 +39,3 @@ export function getTagColor(tags: Map<string, string>, tag: string): string {
   const color: ?string = tags.get(tag);
   return color ? color : '#bbbbbb';
 }
-
-export function findObjetKey(objects, searchValue) {
-  for (const key in objects) {
-    if (objects[key].id === searchValue) {
-      return key
-    }
-  }
-}
-
-export function findObjetById(objects, searchValue) {
-  for (const key in objects) {
-    if (objects[key].id === searchValue) {
```

Comment:
Are `key` and `id` property the same ?
Is it the reason why this function is not needed anymore ?

Reply:
See the removed comment on top of `frontend/src/CreateAnnotationCampaign.js` for more context, I feel the code is more simple and maintainable if `key == id` which is the code I originally created with Objects. However we needed to keep the order given by backend, in order to keep working with Objects Dewi was forced to use such aux functions. However I find it better to use Maps to have both order and key == id so these functions are no longer needed.